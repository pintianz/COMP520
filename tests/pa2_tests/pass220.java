// PA2 pass stmt
class A {
    private void f(){
        a[3]();
    }
}
