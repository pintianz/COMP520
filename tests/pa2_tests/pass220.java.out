======= AST Display =========================
Package
  ClassDeclList [1]
  . ClassDecl
  .   "A" classname
  .   FieldDeclList [0]
  .   MethodDeclList [1]
  .   . (private) MethodDecl
  .   .   VOID BaseType
  .   .   "f" methodname
  .   .   ParameterDeclList [0]
  .   .   StmtList [1]
  .   .   . CallStmt
  .   .   .   IndexedRef
  .   .   .     LiteralExpr
  .   .   .       "3" IntLiteral
  .   .   .     IdRef
  .   .   .       "a" Identifier
  .   .   .   ExprList [0]
=============================================
